admin/app/controllers/authenticated/certifications/scoring-simulation.js

```diff
@@ -0,0 +1,68 @@
+import Controller from '@ember/controller';
+import { action } from '@ember/object';
+import { service } from '@ember/service';
+import { tracked } from '@glimmer/tracking';
+
+export default class ConfigurationController extends Controller {
+  @tracked score = null;
+  @tracked capacity = null;
+  @tracked simulatorReport = null;
+  @tracked errors = [];
+  @service store;
+  @service intl;
+
+  ERRORS = {
+    SCORE: this.intl.t('pages.certifications.scoring-simulation.errors.score'),
+    BOTH_INPUT_FILLED: this.intl.t('pages.certifications.scoring-simulation.errors.both-input-filled'),
+    BOTH_INPUT_EMPTY: this.intl.t('pages.certifications.scoring-simulation.errors.both-input-empty'),
+  };
+
+  @action
+  async onGenerateSimulationProfile(event) {
+    event.preventDefault();
+    this._cleanErrors();
+    this.checkFormValidity();
+    const adapter = this.store.adapterFor('scoring-and-capacity-simulator-report');
```

Comment:
Pour une implémentation similaire, sur la partie rattachement de complémentaire, on était passé via `this.store.query('attachable-target-profile', { searchTerm });`

A voir lequel est mieux, je laisse un pro Ember data étoffer éventuellement

Reply:
A la manière des simulateurs de déroulé d'épreuves, nous ne créons ici pas de modèle Ember, on a voulu aller au plus rapide.
Le `this.store.query` fait appel au modèle du store que nous n'avons pas ici.

Il est tout à fait possible de créer un modèle Ember.
Chose que nous préfèrerions toutefois faire ultérieurement afin de présenter le simulateur le plus rapidement possible aux personnes qui en ont besoin.